src/arrayMethodSort.js

```diff
@@ -5,7 +5,24 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const defaultCompare = (a, b) => {
+      return a.toString() > b.toString();
+    };
+
+    const useFunction = compareFunction || defaultCompare;
+
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let j = i + 1; j < this.length; j++) {
```

Comment:
do you really need nested loop here? you only need to compare neighboring elements